tests/kubernetes/README.md

```diff
@@ -13,30 +13,48 @@ To build this image locally and optionally push to the SkyPilot registry, run:
+     ```
+2. Make sure ports 30000-32767 are open in your node pool VPC's firewall.
 3. Get the kubeconfig for your cluster and place it in `~/.kube/config`:
-```bash
-gcloud container clusters get-credentials <cluster-name> --region <region>
-# Example:
-# gcloud container clusters get-credentials testcluster --region us-central1-c
-```
+   ```bash
+   gcloud container clusters get-credentials <cluster-name> --region <region>
+   # Example:
+   # gcloud container clusters get-credentials testcluster --region us-central1-c
+   ```
 4. Verify by running `kubectl get nodes`. You should see your nodes.
+5. **If you want GPU support**, make sure you install GPU drivers by running:
+   ```bash
+   # If using COS based nodes (e.g., in the example above):
+   kubectl apply -f https://raw.githubusercontent.com/GoogleCloudPlatform/container-engine-accelerators/master/nvidia-driver-installer/cos/daemonset-preloaded.yaml
```

Comment:
Is it possible to add this into some hint message when people try to run `sky launch --gpus V100` on the kubernetes cluster? Currently, it seems only showing timed out when it fails to get the resources on the kubnertest cluster.

Reply:
I've updated `sky check` to return hints for:
1. If GPU drivers/nvidia device plugin is not installed (i.e., `nvidia.com/gpu` resource is not present on the cluster).
2. If GPU types are not labelled correctly (i.e., none of the labels in the `LABEL_FORMATTER_REGISTRY` are available on the cluster. E.g., `cloud.google.com/gke-accelerator` or `skypilot.co/accelerator` labels are not present.).

Example:
```
$ sky check
...
  Kubernetes: enabled
    Hint: Could not detect GPU resources (`nvidia.com/gpu`) in Kubernetes cluster. If this cluster contains GPUs, please ensure GPU drivers are installed on the node. Check if the GPUs are setup correctly by running `kubectl describe nodes` and looking for the nvidia.com/gpu resource. Please refer to the documentation on how to set up GPUs.
...
```

Would this be helpful?